connector/src/main/java/com/google/cloud/bigquery/connector/common/BigQueryClient.java

```diff
@@ -65,6 +64,61 @@ public TableInfo getTable(TableId tableId) {
     return bigQuery.getTable(tableId);
   }
 
+  public boolean tableExists(TableId tableId) {
```

Comment:
Since most of these methods are public, it might be worth adding documentation for methods to make them more understandable for other developers.

Reply:
Good idea!